### ISSUE TITLE
get in-memory

### ISSUE THREAD

**Issue by Rick-Anderson** [Rick-Anderson/VC-EF#1]
@shirhatti @rowanmiller  can you help me get in-memory working.
see 
https://github.com/Rick-Anderson/VC-EF/blob/master/Startup.cs#L16-20

This is the simplest possible MVC 6 app that we will use as our new sample template.

**Comment by rowanmiller** [comment]
That is the right way to set it up, what's not working?

**Comment by Rick-Anderson** [comment]
1. When I add "EntityFramework.InMemory": "7.0.0-rc1-final", 
I get the warning
warning NU1007: Dependency specified was EntityFramework.InMemory >= 7.0.0-rc1-final but ended up with EntityFramework.InMemory 7.0.0-rc2-16701.

Once you add that, the project hangs. You never hit the first line of startup.
That's easy to repro. Create a new empty web app and add     "EntityFramework.InMemory": "7.0.0-rc2-16701",
put a breakpoint on the first line in startup. You never hit it.

**Comment by rowanmiller** [comment]
Yeah that makes sense, you can't mix and match RC1 and RC2 bits (I don't think it is anything EF specific causing the hang, probably something deep inside DNX). You should make sure everything is rc1-final - if you are getting RC2 builds it is probably because you have one of the nightly feeds configured.

**Comment by Rick-Anderson** [comment]
@shirhatti  can you try building/running this? I'm not using RC2 bit and I don't think I have a RC2 feed.
![image](https://cloud.githubusercontent.com/assets/3605364/13656184/cf9e9e90-e608-11e5-82b8-5b5e75616b09.png)

**Comment by Rick-Anderson** [comment]
@shirhatti @danroth27 I'm blocked on View Components until I can get this working. I'll move over to working with Forms.

**Comment by danroth27** [comment]
@Rick-Anderson this builds fine for me: 
![image](https://cloud.githubusercontent.com/assets/1874516/13660373/540e137e-e63e-11e5-9988-3d95732bb7a3.png)

Don't trust the nuget package manager to show you your feeds. Go to the command-line and change to the directory of your app. Use dnvm to make sure 1.0.0-rc1-update1 is the active DNX and then run `dnu feeds list` to see which feeds are being used. You should only have the official nuget.org feed. If you see other feeds use the output from `dnu feeds list` to figure out where they are configured and then fix the configuration.
